README.md

```diff
@@ -65,29 +65,37 @@ future changes by simply running `terraform apply
 | Name | Version |
 |------|---------|
 | terraform | ~> 0.12.0 |
-| aws | ~> 3.0 |
+| aws | ~> 3.38 |
 
 ## Providers ##
 
-| Name | Version |
-|------|---------|
-| aws | ~> 3.0 |
+No providers.
+
+## Modules ##
+
+| Name | Source | Version |
+|------|--------|---------|
+| provisionaccount | github.com/cisagov/provisionaccount-role-tf-module |  |
+
+## Resources ##
+
+No resources.
 
 ## Inputs ##
 
 | Name | Description | Type | Default | Required |
 |------|-------------|------|---------|:--------:|
-| aws_region | The AWS region where the non-global resources for the PCA account are to be provisioned (e.g. "us-east-1"). | `string` | `us-east-1` | no |
-| provisionaccount_role_description | The description to associate with the IAM role that allows sufficient permissions to provision all AWS resources in the PCA account. | `string` | `Allows sufficient permissions to provision all AWS resources in the PCA account.` | no |
-| provisionaccount_role_name | The name to assign the IAM role that allows sufficient permissions to provision all AWS resources in the PCA account. | `string` | `ProvisionAccount` | no |
+| aws\_region | The AWS region where the non-global resources for the PCA account are to be provisioned (e.g. "us-east-1"). | `string` | `"us-east-1"` | no |
+| provisionaccount\_role\_description | The description to associate with the IAM role that allows sufficient permissions to provision all AWS resources in the PCA account. | `string` | `"Allows sufficient permissions to provision all AWS resources in the PCA account."` | no |
+| provisionaccount\_role\_name | The name to assign the IAM role that allows sufficient permissions to provision all AWS resources in the PCA account. | `string` | `"ProvisionAccount"` | no |
 | tags | Tags to apply to all AWS resources provisioned. | `map(string)` | `{}` | no |
```

Comment:
Just for consistency. The variable description would need to be updated as well.

```suggestion
| tags | Tags to apply to all AWS resources created. | `map(string)` | `{}` | no |
```

Reply:
See commit 6808363.